bahmnicore-omod/src/test/java/org/bahmni/module/bahmnicore/web/v1_0/mapper/BahmniFormBuilderObsToTabularViewMapperTest.java

```diff
@@ -250,4 +250,128 @@ public void shouldReturnPivotTableForMultiSelectObs() {
         assertThat(firstRowColumns.get(multiSelectConceptName),
                 containsInAnyOrder(multiSelectFirstObs, multiSelectSecondObs));
     }
+
+    @Test
```

Comment:
Can we add a test to indicate parent and child relationship using row-identifier?

Reply:
Tried to add a test case, but that would be the same as the existing test case since it is private method. Added a comment, hopefully, helps.

Reply:
Updated the test case with a proper name to indicate how add-more rows separated.